packages/components/src/date-time/date/style.scss

```diff
@@ -53,19 +53,29 @@
 		border: none;
 		font-size: $default-font-size;
 		border-radius: $radius-round;
+		background: $white;
+
+		&:hover {
+			color: var(--wp-admin-theme-color);
+		}
 
 		&:focus {
 			box-shadow: inset 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color), inset 0 0 0 #{ $border-width-focus + $border-width } $white;
 			outline: 2px solid transparent; // Shown in Windows 10 high contrast mode.
 		}
 	}
 
+	.CalendarDay__today {
```

Comment:
that is a strange naming convention compared to other components, but I see it matches rest of this scss file - I guess it will get tidied up when it all moves to emotion :-)

Reply:
It's because these classes come from a third party library (`react-dates`).